src/main/java/lotto/domain/Lotto.java

```diff
@@ -0,0 +1,27 @@
+package lotto.domain;
+
+import java.util.Collections;
+import java.util.Set;
+
+public class Lotto {
+
+    private static final int LOTTO_NUMBERS_SIZE = 6;
+    private static final String ERROR_LOTTO_NUMBER = "로또 번호는 6개여야 합니다.";
+    private Set<LottoNumber> lottoNumber;
```

Comment:
`상수`와 `상태`(= 인스턴스 변수)는 줄 바꿈해서 가독성을 높여주세요! 🙏 
또한 상태 값은 Lotto의 상태 값은 `6개(복수)의 로또 숫자`이므로 네이밍을 복수형으로 사용하는게 더 의미가 명확한 것 같아요. 👀 

```suggestion
    private static final String ERROR_LOTTO_NUMBER = "로또 번호는 6개여야 합니다.";
    
    private final Set<LottoNumber> lottoNumbers;
```

Reply:
반영하였습니다 👍